src/app/modules/payments/stripe.service.ts

```diff
@@ -1009,15 +1011,155 @@ export const verifyPaymentStatusWithStripe = (
     })
 }
 
-export const generatePaymentInvoice = (
+const getPaymentProofObjectPath = (payment: IPaymentSchema) => {
+  return payment.formId + '/' + payment._id + '.pdf'
+}
```

Comment:
/nit: feels like something which belongs in `.utils`

Reply:
Yep agreed, added in 69002e5e1